### ISSUE TITLE
Problem with Edimax EW-7811UN (wifi)

### ISSUE THREAD

**Issue by Yamakaky** [recalbox/recalbox-os#97]
I use an Edimax EW-7811UN with my raspberry pi B, and I can't connect to the wifi network. I checked my password several times. Any idea ?
Btw, it would be more usefull to display a list of available wifi networks, than input a BSID + password.

**Comment by hstampfl** [comment]
have you also tried to configure the wifi stuff in /recalbox/share/system/recalbox.conf file.
I have running two EW-7811UN  running

**Comment by hstampfl** [comment]
those ones I mentioned whre updated versions. On a new installed recalbox I do also get configuration errors.

**Comment by Yamakaky** [comment]
No, I will try at home in one hour. Thanks.

**Comment by hstampfl** [comment]
ah, try , the newest version, it has a sleepmode fix included. Perhaps this helps

**Comment by Yamakaky** [comment]
I tried with the last version, nope.
I can't edit recalbox.conf : it's on ext4 which is not readable by macos.

**Comment by digitalLumberjack** [comment]
recalbox.conf is on a fat32 partition.

**Comment by digitalLumberjack** [comment]
on the share partition `system`directory

**Comment by Yamakaky** [comment]
btw, why can't we modify it in recovery mode ? I don't know raspbian, is it possible ?

**Comment by Yamakaky** [comment]
No, it still doesn't work... The wifi menu reports a problem in the network configuration.

**Comment by hstampfl** [comment]
can you please provide dmesg from your  EW-7811UN.
mine looks like this:
[    2.817306] usb 1-1.4: new high-speed USB device number 5 using dwc_otg
[    2.918517] usb 1-1.4: New USB device found, idVendor=7392, idProduct=7811
[    2.918540] usb 1-1.4: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[    2.918551] usb 1-1.4: Product: 802.11n WLAN Adapter
[    2.918561] usb 1-1.4: Manufacturer: Realtek
[    2.918572] usb 1-1.4: SerialNumber: 00e04c000001
.
.
.
[    3.521195] usbcore: registered new interface driver rtl8192cu

**Comment by Yamakaky** [comment]
On my archlinux, it shows : 

```
[32577.250164] usb 3-1.3: new high-speed USB device number 11 using ehci-pci
[32577.349678] rtl8192cu: Chip version 0x10
[32577.435890] rtl8192cu: MAC address: 80:1f:02:e3:80:00
[32577.435901] rtl8192cu: Board Type 0
[32577.436131] rtl_usb: rx_max_size 15360, rx_urb_num 8, in_ep 1
[32577.436202] rtl8192cu: Loading firmware rtlwifi/rtl8192cufw_TMSC.bin
[32577.436359] ieee80211 phy2: Selected rate control algorithm 'rtl_rc'
[32577.457140] rtl8192cu 3-1.3:1.0 wlp0s26f7u1u3: renamed from wlan0
```

**Comment by hstampfl** [comment]
fyi, I do have an RPi 1 B+ with a fresh installation of recalboxOS where I do get the same error. But on a RPi2 with an updated installation of recalboxOS the stick is working. -  I would like to verify if we have the  wifi dongle with the same chipset. - This was the reason I asked about it

On my working RPi I do see the wpa_supplicant.conf on two diffrent places. 
/etc/wpa_supplicant.conf and
/etc/wpa_supplicant/wpa_supplicant.conf

try to add the above files manually with following lines (between the dashes):
---------
ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=root

network={
        ssid="put_here_your_SSID"
        psk="put_here_your_WifiPassword"
}
---------

and last but not least reboot and show me your result... (I do have the not working RPi not with me, that's why I ask you to try this.)

**Comment by Yamakaky** [comment]
wpa_supplicant.conf is correct... However, /etc/wpa_supplicant doesn't exist.
What do you mean by "show me the result"?

**Comment by wh1stler** [comment]
Want to throw my hat in here as well with the same issue - Tried the Vilros adapter that is based on the realtek chipset and now the Edimax with realtek and cannot get either to connect.

**Comment by mxjoe** [comment]
Is the WiFi dongle detected as "wlan0"? Please send us the output of "ifconfig -a". If it is not detected it might be due to a missing firmware file as mentioned in issue #230 .

**Comment by krustowsky** [comment]
Plug in the WiFi-Receiver to the upper (or another) USB-Slot. Worked for me with the Edimax.

**Comment by nadenislamarre** [comment]
fixed in 4.1
